packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useModifiersStore.js

```diff
@@ -72,7 +72,7 @@ export function useModifiersStore() {
    * @param {string} name - The name of the modifier to toggle
    */
   const baseHandleToggle = (name) => {
-    if (modifiers[name]) {
+    if (modifiers && modifiers[name]) {
```

Comment:
I believe this would work

```suggestion
    if (modifiers?.[name]) {
```

Reply:
The problem I think is that when the Editor is not in focus the first time click desn't work, I am trying to find a way to do it